cassandra/src/main/scala/filodb.cassandra/Util.scala

```diff
@@ -57,7 +57,7 @@ object Util {
       Futures.addCallback(lf, new FutureCallback[A] {
         def onFailure(t: Throwable): Unit = promise failure t
         def onSuccess(result: A): Unit = promise success result
-      })
+      }, GlobalScheduler.globalImplicitScheduler)
```

Comment:
Is this needed as part of this PR?

Reply:
Yes, needed. Kamon upgrade caused guava upgrade. I have changed code to accept implicit scheduler as arg